app/models/dataset_record/darwin_core/taxon.rb

```diff
@@ -57,23 +57,25 @@ def import(dwc_data_attributes = {})
         year = nil
 
         # split authorship into name and year
-        if nomenclature_code == :iczn
-          if (authorship_matchdata = authorship.match(/\(?(?<author>.+),? (?<year>\d{4})?\)?/))
+        if authorship.present?
+          if nomenclature_code == :iczn
+            if (authorship_matchdata = authorship.match(/\(?(?<author>.+),? (?<year>\d{4})?\)?/))
```

Comment:
Are you specifically looking for original combination author of new combination names or you'd rather need authorship of any name? (Question is because it looks you looks for authorship in parentheses?)

Reply:
We're looking for any authorship, the parentheses are optional

Reply:
We should be leveraging `Biodiversity::Parser` authorship info if it's available, but fixing the nil error was my priority.

Reply:
Ah sorry, missed that parens are optionally matched.